projects/lookup/src/modules/lookup/lookup-show-more-modal.component.ts

```diff
@@ -17,6 +17,7 @@ import { SkyLookupShowMoreNativePickerContext } from './types/lookup-show-more-n
  * Internal component to implement the native picker.
  */
 @Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
   selector: 'skyux-lookup-show-more-modal',
```

Comment:
That was an existing component that was giving a warning, so I suppressed it instead of making a potentially breaking change for consumers. The show more async modal component on which this is based does have the correct selector.